api/src/main/java/org/openmrs/OrderAttributeType.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import org.openmrs.attribute.AttributeType;
+import org.openmrs.attribute.BaseAttributeType;
+
+/**
+ * A user-defined extension to the {@link Order} class.
+ * @see AttributeType
+ * @since 2.4.0
```

Comment:
Is the above since correct?

Reply:
@dkayiwa, The docs stated this approach started with 2.4.0, do I need to use the current build 2.5.0?

Reply:
@miirochristopher which docs are they?